python/graphstorm/model/utils.py

```diff
@@ -311,9 +312,112 @@ def _exchange_node_id_mapping(local_rank, world_size, device,
         th.distributed.all_to_all(gather_list, data_tensors)
     return gather_list[0]
 
-def save_embeddings(model_path, embeddings, local_rank, world_size,
+def distribute_nid_map(embeddings, local_rank, world_size, 
```

Comment:
is this really a local rank or a global rank?

Reply:
i believe this is global rank. @classicsong

Reply:
global rank.
Can you rename it to rank?